angular-workspace/projects/ni/nimble-angular/build/generate-icons/source/index.js

```diff
@@ -51,10 +51,11 @@ for (const key of Object.keys(icons)) {
 
     const directiveFileContents = `${generatedFilePrefix}
 import { Directive } from '@angular/core';
-import type { ${className} } from '@ni/nimble-components/dist/esm/icons/${directoryName}';
+import { type ${className}, icon${pascalCase(iconName)}Tag } from '@ni/nimble-components/dist/esm/icons/${directoryName}';
```

Comment:
nit: for readability let's add a variable above for `icon${pascalCase(iconName)}Tag` with a comment like the others on line 44-49.